keydata.go

```diff
@@ -103,53 +113,16 @@ func (e *PlatformDeviceUnavailableError) Unwrap() error {
 // DiskUnlockKey is the key used to unlock a LUKS volume.
 type DiskUnlockKey []byte
 
-// AuxiliaryKey is an additional key used to modify properties of a KeyData
+// PrimaryKey is an additional key used to modify properties of a KeyData
 // object without having to create a new object.
-type AuxiliaryKey []byte
+type PrimaryKey []byte
```

Comment:
for future consideration: this kind of rename would have been best done has its own PR

Reply:
as I wrote in the overall comment, this actual would be useful